### PR TITLE
secrets docs

### PR DESCRIPTION
**Pull request by edenlightning**
Update secrets documentation

cc @borda @rohitgr7 @Felonious-Spellfire

### CODE REVIEW

**Review by yurijmikhalevich — APPROVED** [review]

File: docs/source-app/glossary/secrets.rst [hunk 1]
```diff
@@ -4,55 +4,56 @@
...
-***************
+************
+Add a secret
+************
 
-When a Lightning App (App) **runs in the cloud**, a Secret can be exposed to the App using environment variables.
-The value of the Secret is encrypted in the Lightning.ai database, and is only decrypted and accessible to
-LightningFlow (Flow) or LightningWork (Work) processes in the cloud (when you use the ``--cloud`` option running your App).
+Add the secret to your profile on lightning.ai.
+Log in to your lightning.ai account > **Profile** > **secrets** tab > click the **+New** button.
```
Comment (yurijmikhalevich) [inline docs/source-app/glossary/secrets.rst:18]:
```suggestion
Log in to your lightning.ai account > **Profile** > **Secrets** tab > click the **+New** button.
```

File: docs/source-app/glossary/secrets.rst [hunk 2]
```diff
@@ -4,55 +4,56 @@
...
+Provide a name and value to your secret, for example, name could be "github_api_token".
 
-----
-
-**********************
-What were we thinking?
-**********************
+.. note::
+    Secret names must start with a letter and can only contain letters, numbers, dashes, and periods. The Secret names must comply with `RFC1123 naming conventions <https://www.rfc-editor.org/rfc/rfc1123>`_. The Secret value has no restrictions.
```
[the remark this answers is not in the document]
Comment (edenlightning) [reply docs/source-app/glossary/secrets.rst:22]: Thats sphinx: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

File: docs/source-app/glossary/secrets.rst [hunk 3]
```diff
@@ -4,55 +4,56 @@
 Encrypted Secrets
 #################
 
-Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and are decrypted at runtime.
+Encrypted Secrets allow you to pass private data to your apps, like API keys, access tokens, database passwords, or other credentials, in a secure way without exposing bthem in your code.
```
Comment (awaelchli) [inline docs/source-app/glossary/secrets.rst:7]:
```suggestion
Encrypted Secrets allow you to pass private data to your apps, like API keys, access tokens, database passwords, or other credentials, in a secure way without exposing them in your code.
```

File: docs/source-app/glossary/secrets.rst [hunk 4]
```diff
@@ -4,55 +4,56 @@
...
 
-*********************
-Use Encrypted Secrets
-*********************
+************
+Use a secret
+************
 
-To use Encrypted Secrets:
+1. Add an environment variables to your app to read the secret. For example, add an "api_token" environment variable:
```
Comment (awaelchli) [inline docs/source-app/glossary/secrets.rst:37]: plural - singular conflict

**Review by awaelchli — APPROVED** [review]

**Review by Borda — COMMENTED** [review]
pls, add this page also to lit docs

**Comment by awaelchli** [comment]
@Borda Eden probably does not know how to do that. Should it be a symlink?

**Comment by awaelchli** [comment]
@Borda Does it need to be a symlink or what do you mean by that?

**Comment by Borda** [comment]
> @Borda Does it need to be a symlink or what do you mean by that?

fixed :)

**Review by Borda — APPROVED** [review]

**Pull request merged**